File: documentation/user/en/query/filtering/examples/comparable/attribute-less-than-equals.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeLessThanEquals("battery-capacity", 125)
				),
				require(
					entityFetch(
						attributeContent("code", "battery-capacity")
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/filtering/examples/comparable/attribute-less-than-equals.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeLessThanEquals("battery-capacity", 125)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code", "battery-capacity")
        		)
        	)
        )
	)
);